Document/0x06a-Platform-Overview.md

```diff
@@ -12,20 +12,21 @@ Like the Apple desktop operating system macOS (formerly OS X), iOS is based on D
 
 However, iOS apps run in a more restricted environment than their desktop counterparts do. iOS apps are isolated from each other at the file system level and are significantly limited in terms of system API access.
 
-To protect users from malicious applications, Apple restricts and controls access to the apps that are allowed to run on iOS devices. The Apple App store is the only official application distribution platform. There developers can offer their apps and consumers can buy, download, and install apps. This distribution style differs from Android, which supports several app stores and sideloading (installing an app on your iOS device without using the official App store).
+To protect users from malicious applications, Apple restricts and controls access to the apps that are allowed to run on iOS devices. Apple's App Store is the only official application distribution platform. There developers can offer their apps and consumers can buy, download, and install apps. This distribution style differs from Android, which supports several app stores and sideloading (installing an app on your iOS device without using the official App Store, 
+commonly via USB).
 

```

Comment:
We should not forget about installing apps with enterprise certificates as well right? See https://developer.apple.com/programs/enterprise/

Reply:
I agree, but this method is not covered in detailed yet in the MSTG, so this is why I avoided to mention it yet.